var/plugins/barmanager.py

```diff
@@ -1100,6 +1109,158 @@ def hGetLastVote(source, user, params, checkOnly):
         spads.slog("Unhandled exception: " + str(sys.exc_info()
                    [0]) + "\n" + str(traceback.format_exc()), 0)
 
+def hMinRatingLevel(source, user, params, checkOnly):
```

Comment:
All those commands definitions are pretty verbose IMHO, and don't really add much clarity. I would suggest doing a wrapped that creates a new handler, something like:

```py
def getGenericTeiserverStringCommandHandler(cmd, validRegex):
  def handler(source, user, params, checkOnly):
    spads.slog("User %s called command %s with parameter(s) \"%s\"" % (
                user, cmd, ','.join(params)), DBGLEVEL)
    try:
      return genericTeiserverStringCommand(source, user, params, checkOnly, cmd, validRegex)
    except Exception as e:
      spads.slog("Unhandled exception: " + str(sys.exc_info()[0]) + "\n" + str(traceback.format_exc()), 0)

  return handler
```

then at call site, just e.g.

```py
spads.addSpadsCommandHandler({'rename': getGenericTeiserverStringCommandHandler('rename', "^[a-zA-Z0-9_\-\[\] \<\>\+\|:]+$")})
```